app/controllers/group_assignment_invitations_controller.rb

```diff
@@ -122,6 +128,16 @@ def create_group_assignment_repo(selected_group: group, new_group_title: nil)
   # rubocop:enable Metrics/AbcSize
   # rubocop:enable MethodLength
 
+  ## Resource Helpers
+
+  def group
+    repo_access = current_user.repo_accesses.find_by(organization: organization)
+    return unless repo_access.present? && repo_access.groups.present?
+
```

Comment:
This method isn't new, it was just moved to a different spot in the file for organization purposes.